addons/func_godot/src/map/func_godot_map.gd

```diff
@@ -82,6 +95,11 @@ func verify_and_build() -> void:
 	else:
 		emit_signal("build_failed")
 
+func _on_reimport(resources: PackedStringArray) -> void:
+	for resource in resources:
+		if resource == local_map_file:
```

Comment:
This should also be checking global_map_file

Reply:
Not possible, Godot won't notify about a file change that *isn't int its directory*. If the map *is* in the directory, I don't see why the global field is being used at all.
If I'm misunderstanding the use case, please let me know.

Reply:
> Not possible, Godot won't notify about a file change that isn't inside the project directory. If the map is in the directory, I don't see why the global field is being used at all.

I'm more concerned about user error here. global_map_file is supposed to override local_map_file, that logic should be here too. Additionally, it catches cases where people are using global_map_file field for local maps. View the class reference on FuncGodotMap for this overriding, specifically on the properties